sql/src/main/java/io/crate/planner/operators/WindowAgg.java

```diff
@@ -70,9 +70,11 @@
         for (WindowFunction windowFunction : windowFunctions) {
             WindowDefinition windowDefinition = windowFunction.windowDefinition();
             if (!windowDefinition.windowFrameDefinition().equals(WindowDefinition.UNBOUNDED_PRECEDING_CURRENT_ROW) &&
-                !windowDefinition.windowFrameDefinition().equals(WindowDefinition.CURRENT_ROW_UNBOUNDED_FOLLOWING)) {
-                throw new UnsupportedFeatureException("Only unbounded preceding -> current row and current row -> " +
-                                                      "unbounded following frame definitions are supported");
+                !windowDefinition.windowFrameDefinition().equals(WindowDefinition.CURRENT_ROW_UNBOUNDED_FOLLOWING) &&
+                !windowDefinition.windowFrameDefinition().equals(WindowDefinition.UNBOUNDED_PRECEDING_UNBOUNDED_FOLLOWING)) {
+                throw new UnsupportedFeatureException(
```

Comment:
Could we remove the restrictions here alltogether and instead rely on the window frame type implementation resulting in a runtime failure?

Reply:
That would postpone the validation until execution time (at the batch iterator level). I think that's too late (esp in a distributed execution)

Reply:
What is it too late for? It is a bit more expensive, but I don't think we've to optimize the error path for statements which are unsupported. 

It is unlikely that unsupported statements will be invoked often by applications. 

Either way, it is also fine for me to keep it as is. (But tests are failing)

Reply:
Too late in that we already collected and partitioned the necessary data and the execution of the window agg projector will fail on every node involved. It's the ssl tests that fail ( SslReqHandlerIntegrationTest )